### PR TITLE
issue-2-view-entries - Ken Kieu

### PR DESCRIPTION
**Pull request by kenkieu**
Completed issue-2-view-entries.
code-journal-issue-2

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -20,8 +20,102 @@ function handleSubmit(event) {
   data.nextEntryId++;
   data.entries.unshift(entry);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
+  var instantEntry = entryTemplate(data.entries[0]);
+  $ul.prepend(instantEntry);
   $form.reset();
+  switchView($entriesPage.getAttribute('data-view'));
+  data.view = 'entries';
```
Comment (codingcodymiller) [inline js/main.js]: You should set `data.view` in `switchView` when you find the correct view, so you don't need to remember to do it every time you switch views.

File: js/main.js [hunk 2]
```diff
@@ -20,8 +20,102 @@ function handleSubmit(event) {
   data.nextEntryId++;
   data.entries.unshift(entry);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
+  var instantEntry = entryTemplate(data.entries[0]);
+  $ul.prepend(instantEntry);
   $form.reset();
+  switchView($entriesPage.getAttribute('data-view'));
```
Comment (codingcodymiller) [inline js/main.js]: You can just do `switchView("entries")`

File: js/main.js [hunk 3]
```diff
@@ -20,8 +20,102 @@ function handleSubmit(event) {
...
+    <div class="column-half">
+      <h2>Coffee</h2>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ratione dicta voluptates reprehenderit
+        alias
+        nobis atque aperiam! Nobis cumque amet molestiae dicta voluptates reprehenderit alias
+        nobis atque aperiam!</p>
+    </div>
+  </div>
+</li>
+*/
```
Comment (codingcodymiller) [inline js/main.js]: I would put this inside your `entryTemplate` function so it's clear that it is the structure that function is creating.

File: js/main.js [hunk 4]
```diff
@@ -20,8 +20,102 @@ function handleSubmit(event) {
...
+  switchView($entriesPage.getAttribute('data-view'));
+  data.view = 'entries';
+});
+
+$newBtn.addEventListener('click', function (event) {
+  switchView($formPage.getAttribute('data-view'));
+  data.view = 'entry-form';
+});
+
+switchView(data.view);
```
Comment (codingcodymiller) [inline js/main.js]: I would move this inside `handleDOMContentLoaded`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by kenkieu** [comment]
Fixed issue of also updating data.view when the incorrect view was being displayed.

refresh-bug-fixed

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**